presto-main/src/test/java/com/facebook/presto/type/TestTimeWithTimeZone.java

```diff
@@ -233,4 +228,82 @@ public void testCastToSlice()
         assertFunction("cast(TIME '03:04:05 +07:09' as varchar)", VARCHAR, "03:04:05.000 +07:09");
         assertFunction("cast(TIME '03:04 +07:09' as varchar)", VARCHAR, "03:04:00.000 +07:09");
     }
+
+    @Test
+    public void testCastFromSlice()
```

Comment:
It's used around here though. That's why I followed this overwhelming convention ;-)
Should I change names of all such tests (testCastFromSlice and testCastToSlice in all DateTime tests)?